crates/ruff_server/src/server.rs

```diff
@@ -354,3 +363,56 @@ impl SupportedCodeAction {
         .into_iter()
     }
 }
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub(crate) enum SupportedCommand {
+    Debug,
+    Format,
+    FixAll,
+    OrganizeImports,
+}
```

Comment:
I renamed this from `Command` to `SupportedCommand` to match `SupportedCodeAction` and also moved it from the `execute_command.rs` module. Feel free to disagree on anything.